### ISSUE TITLE
Dialogs on Linux enviroment show at top left corner before getting centered

### ISSUE THREAD

**Issue by abhinayagarwal** [javafxports/openjdk-jfx#129]
Alert/Dialog on Linux with JavaFX 11 doesn't show at center. Instead, they show at (0, 0) and then gets centered. This might be Ubuntu specific bug.

Here is a video showing Dialog in Java 8 vs Java 11:

<a target="_blank" href="https://drive.google.com/file/d/10fVL-JZCZhIe_CXAr5fRRaaTAqa-co6U/view?usp=sharing">Video of issue</a>

Environment Details
----
Ubuntu 18.04 LTS
OpenJDK Runtime Environment 18.9 (build 11-ea+21)
OpenJFX-11-ea+17_linux-x64

Sample
----
The sample running in the video comes from this [Github repository](https://github.com/JonathanGiles/fx-ui-controls/blob/master/src/main/java/org/modernclients/ch3/samples/DialogsDemo.java).

**Comment by abhinayagarwal** [comment]
Related to #217

**Comment by DJViking** [comment]
I have not noticed this on OpenSUSE Leap 15. 
The window pop up right in the center of my screen in an instant.
NVIDIA graphics with NVIDIA proprietary drivers.

Running my modular test application on Java 11 and JavaFX 11.
https://github.com/DJViking/FxMovies.git

I did however noticed for a split second that a dialog I opened was in the upper left corner. It was hardly noticeable.

I have also tried the Sample application mention in the OP. Running with Java 11 and JavaFX 11 I can not notice this effect.
